test/test_dipolarbackground.py

```diff
@@ -142,7 +142,6 @@ def test_multipath_raw():
     for p in range(len(lam)):
             Bref = Bref*Bmodel((t-T0[p]),lam[p])
             Bnorm = Bnorm*Bmodel(-T0[p],lam[p])

```

Comment:
Remove `Bnorm`

Reply:
Oops, seems like I missed a couple of those. Should be removed now.